### PR TITLE
Node.js docs follow-ups and general docs cleanup

### PR DESCRIPTION
**Pull request by gj**
- [x] Update Node.js REPL docs

### CODE REVIEW

File: docs/using/examples/context.rst [hunk 1]
```diff
@@ -54,9 +54,13 @@ variables.
            :caption: :fab:`java` Env.java
```
Comment (gj) [inline docs/using/examples/context.rst:54]: as a meta comment, I don't think `Context` is a very descriptive name for this docs page

File: docs/more/dev-tools/repl.rst [hunk 2]
```diff
@@ -186,15 +219,15 @@ plus ``oso``, and then use the ``Oso.repl()`` API method to start the REPL:
...
-            const { Expense, User } = require("./models");
-            const { Oso } = require("oso");
+            const { Expense, User } = require('./models');
+            const { Oso } = require('oso');
 
             const oso = new Oso();
             oso.registerClass(Expense);
             oso.registerClass(User);
-            await oso.repl();
+            oso.repl();
```
Comment (dhatch) [inline docs/more/dev-tools/repl.rst:233]: Why did you remove `await` here?  This is still an async method.

Comment (gj) [reply docs/more/dev-tools/repl.rst:233]: Because if someone copies this code into a script and runs `node path/to/that/script.js` it'll work either way, but if someone opens up the Node REPL and copies and pastes these lines in it'll blow up if they try to use a top-level `await` (unless they started Node with the `--experimental-repl-await` feature) but it'll work fine if they skip the `await`.
